Libs/DICOM/Widgets/ctkDICOMTableView.cpp

```diff
@@ -615,6 +675,92 @@ void ctkDICOMTableView::addSqlWhereCondition(const QString column, const QString
   d->sqlWhereConditions.insert(column, values);
 }
 
+//------------------------------------------------------------------------------
+void ctkDICOMTableView::removeSqlWhereCondition(const QString column)
```

Comment:
```suggestion
void ctkDICOMTableView::removeSqlWhereCondition(const QString& column)
```

Reply:
Just out of curiosity: why is it important to pass a column name (which is usually not longer than a dozen characters) by pointer not value? I'll change these anyway just wondering...

Reply:
Matter of consistency